filters/ThirdParty/filter_216_PolskiFiltr/metadata.json

```diff
@@ -1,6 +1,6 @@
 {
   "filterId": 216,
-  "name": "Adblock & uBlock polish filter",
+  "name": "Official Polish filters for AdBlock, uBlock Origin & AdGuard",
   "description": "polski filtr",
```

Comment:
Do we need to change the description?

Reply:
I'm not sure, but other filter lists have similar description, for example:
https://github.com/AdguardTeam/FiltersRegistry/blob/master/filters/ThirdParty/filter_203_hufilter/metadata.json#L4
and
https://github.com/AdguardTeam/FiltersRegistry/blob/master/filters/ThirdParty/filter_201_CzechList/metadata.json#L4

Reply:
We usually just use the description from the filter file

Reply:
So what should I add here?
This? - `Oficjalne polskie filtry do AdBlocka, uBlocka Origin i AdGuarda`
https://github.com/MajkiIT/polish-ads-filter/blob/master/polish-adblock-filters/adblock.txt#L3

Reply:
Yeah, that's right

Reply:
Ok, I changed description and homepage addresses which was mentioned here https://github.com/AdguardTeam/FiltersRegistry/issues/40#issuecomment-381832874 and here https://github.com/AdguardTeam/FiltersRegistry/issues/40#issuecomment-383333269